tests/FixerDefinition/VersionedCodeSampleTest.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\FixerDefinition;
+
+use PhpCsFixer\FixerDefinition\VersionedCodeSample;
+
+/**
+ * @author Andreas Möller <am@localheinz.com>
+ *
+ * @internal
+ */
+final class VersionedCodeSampleTest extends \PHPUnit_Framework_TestCase
+{
+    public function testImplementsVersionedCodeSampleInterface()
+    {
+        $reflection = new \ReflectionClass('PhpCsFixer\FixerDefinition\VersionedCodeSample');
+
+        $this->assertTrue($reflection->implementsInterface('PhpCsFixer\FixerDefinition\VersionedCodeSampleInterface'));
```

Comment:
question: what is a benefit of this test ?
if concrete class would not follow interface, the other tests would fail